consensus/src/dag/reconstruction/mod.rs

```diff
@@ -76,6 +77,23 @@ impl<U: Unit> UnitWithParents for ReconstructedUnit<U> {
     }
 }
 
+impl<D: Data, H: Hasher, K: MultiKeychain> From<ReconstructedUnit<Signed<FullUnit<H, D>, K>>>
+    for OrderedUnit<D, H>
+{
+    fn from(unit: ReconstructedUnit<Signed<FullUnit<H, D>, K>>) -> Self {
+        let parents = unit.parents;
+        let hash = unit.unit.hash();
+        let (unit, data) = unit.unit.into_signable().into();
```

Comment:
With the changes above this should probably be changed into `unit.unpack().as_signable().data().clone()` and the `From` impl can be removed.

Reply:
changed